parts/masteroutputs.t

```diff
@@ -6,7 +6,7 @@
       "value": ""
 {{end}}
     }
-{{if AnyAgentUsesAvailabilitySets}}
+{{if HasVMASAgentPool}}
```

Comment:
renamed `AnyAgentUsesAvailabilitySets` to `HasVMASAgentPool` to match the VMSS equivalent